src/zenml/integrations/integration.py

```diff
@@ -82,6 +83,11 @@ def check_installation(cls) -> bool:
             )
             return False
 
+    @classmethod
+    def check_system(cls) -> None:
```

Comment:
Looking at this again, perhaps `update_system_specific_requirements` or something similar might be a more descriptive name? @schustmi any suggestions?